Automattic/fresh-data
# Update package.lock files

This updates package.lock files across the board.

Nothing needed for review here, just build to verify.

To Test:

Just run the steps in the `.travis.yml` file to ensure a local build.
Optional: Test example apps.
